spec/integration/rspec_spec.rb

```diff
@@ -1,6 +1,9 @@
 require 'spec_helper'
 
-describe TestController, 'rspec integration', :integration => true, :target => 'rspec' do
+describe TestController, 'rspec integration', integration: 'true' do
+
+  # TODO: need to rewrite all of this to support Rails 5
+  #
   # Hack to allow us to include the ActionController::TestCase::Behaviour module
```

Comment:
this is a big todo, there's a lot of ugly hacks in this test to mock on TestController, I think I need to switch everything to use `ActionDispatch::Integration` if I want these 4 tests to work in Rails 5x